tests/adapter/conftest.py

```diff
@@ -59,7 +59,7 @@ def parsed_gene():
         "hgnc_id": 1,
         "hgnc_symbol": "AAA",
         "ensembl_id": "ENSG1",
-        "chrom": "1",
+        "chromosome": "1",
```

Comment:
Hm, yes, I think this is correct - the `chrom` should all have been temporary shorthands. I will say it is a bit confusing to have `chrom` for `Exon`, `HgncTranscript` etc, but we are in a small majority of "chromosome" for storage so go.

Reply:
It gets saved as chromosome in the database, it was just the passages before, it's first chromosome, then [becomes chrom](https://github.com/Clinical-Genomics/scout/blob/415bfff6d84eb76765ba5599065100d17029843f/scout/build/genes/hgnc_gene.py#L100), to be [converted again](https://github.com/Clinical-Genomics/scout/blob/415bfff6d84eb76765ba5599065100d17029843f/scout/models/hgnc_map.py#L126) in chromosome in the model. Not confusing at all!

Reply:
Exactly, keeps you on your toes, doesn’t it! 😆 There may be something to different names names for some things in disk/db/mem by need or to quickly tell
things that are further decorated in memory apart. But different names to member vars that store the same is not it.